### PR TITLE
Create service-proxy.md

### LINKED ISSUE #2686: [hi] Localize Service Proxy for Hindi
Localize Service Proxy Site for Hindi localization folks from https://glossary.cncf.io/service-proxy/

### PR DESCRIPTION
**Pull request by Aryan-Singla**
### Describe your changes
Localize the service proxy in Hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2686 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias. ci "commit -s"`.

### CODE REVIEW

File: content/hi/service-proxy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+शीर्षक: सर्विस प्रॉक्सी (Service Proxy)
+status: Completed
+category: टैकनोलजी
+tags: ["नेटवर्किंग"]
+---
+
+सर्विस प्रॉक्सी किसी दिए गए [सर्विस](/service/) से या उससे आने वाले ट्रैफ़िक को रोकता है,
+उसमे फिर कुछ तर्क लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:9]:
```suggestion
उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है
```

File: content/hi/service-proxy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+यह अनिवार्य रूप से एक " मध्यस्थ " के रूप में कार्य करता है जो नेटवर्क ट्रैफ़िक के बारे में जानकारी एकत्र करता है और/या उस पर नियम लागू करता है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:10]:
```suggestion
उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है
यह अनिवार्य रूप से एक "मध्यस्थ" के रूप में कार्य करता है जो नेटवर्क ट्रैफ़िक के बारे में जानकारी एकत्र करता है और उस पर नियम लागू करता है।
```

File: content/hi/service-proxy.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## यह समस्या का समाधान करता है
+
+सर्विस से सर्विस संचार (उर्फ नेटवर्क ट्रैफ़िक) का ट्रैक रखने के लिए और
+संभावित रूप से इसे परिवर्तन या रेडिरेक्ट करने के लिए, हमें डेटा एकत्र करने की आवश्यकता है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:15]:
```suggestion
संभावित रूप से इसे परिवर्तित या पुन: निर्देशित करने के लिए, हमें डेटा एकत्र करने की आवश्यकता होती है।
```

File: content/hi/service-proxy.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को "एक्सटेरनलिज़" करने की अनुमति देती है।
+अब इसे ऐप्स के भीतर रहने की आवश्यकता नहीं है।
+इसके बजाय, अब इसे प्लेटफ़ॉर्म परत (जहां आपके ऐप्स चलते हैं) में एम्बेड किया गया है।
+
+सर्विस के बीच द्वारपाल के रूप में कार्य करते हुए, प्रॉक्सी यह जानकारी प्रदान करते हैं कि किस प्रकार का संचार हो रहा है।
+अपनी अंतर्दृष्टि के आधार पर, वे यह निर्धारित करते हैं कि किसी विशेष अनुरोध को कहां भेजना है या इसे पूरी तरह से अस्वीकार करना है।
+
+प्रॉक्सी महत्वपूर्ण डेटा एकत्र करते हैं, रूटिंग का प्रबंधन करते हैं (सर्विस के बीच ट्रैफ़िक को समान रूप से फैलाना या यदि कुछ सेवाएँ ख़राब हो जाती हैं तो पुनः रूट करना),
+एन्क्रिप्ट कनेक्शन, और कैश सामग्री (संसाधन खपत को कम करना)।–––
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:28]:
```suggestion
एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को "एक्सटर्नालिजे" करने की अनुमति देती है।
अब इसे ऐप्स के भीतर रहने की आवश्यकता नहीं होगी।
इसके बजाय, अब इसे प्लेटफ़ॉर्म परत (जहां आपके ऐप्स चलते हैं) में एम्बेड किया जायेगा।

सर्विस के बीच द्वारपाल के रूप में कार्य करते हुए, प्रॉक्सी यह जानकारी प्रदान करती है कि किस प्रकार का संचार हो रहा है।
अपनी अंतर्दृष्टि के आधार पर, वे यह निर्धारित करती है कि किसी विशेष अनुरोध को कहां भेजना है या इसे पूरी तरह से अस्वीकार करना है।

प्रॉक्सी महत्वपूर्ण डेटा एकत्र करती है, रूटिंग का प्रबंधन करती है (सर्विस के बीच ट्रैफ़िक को समान रूप से फैलाना या यदि कुछ सेवाएँ ख़राब हो जाती हैं तो पुनः रूट करना),
इंक्रिप्टेड कनेक्शन, और कैश मैमोरी (संसाधन खपत को कम करना)।–––
```

**Comment by Aryan-Singla** [comment]
Hello @aj11anuj if you have bandwidth then Please check this localization is there any need for improvement

File: content/hi/service-proxy.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+शीर्षक: सर्विस प्रॉक्सी (Service Proxy)
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:2]:
```suggestion
title: सर्विस प्रॉक्सी (Service Proxy)
```

File: content/hi/service-proxy.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+status: Completed
+category: टैकनोलजी
+tags: ["नेटवर्किंग"]
+---
+
+सर्विस प्रॉक्सी किसी दिए गए [सर्विस](/service/) से या उससे आने वाले ट्रैफ़िक को रोकता है,
+उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:9]:
```suggestion
उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है।
```

File: content/hi/service-proxy.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+यह अनिवार्य रूप से एक "मध्यस्थ" के रूप में कार्य करता है जो नेटवर्क ट्रैफ़िक के बारे में जानकारी एकत्र करता है और उस पर नियम लागू करता है।
+
+## यह समस्या का समाधान करता है
+
+सर्विस से सर्विस संचार (उर्फ नेटवर्क ट्रैफ़िक) का ट्रैक रखने के लिए और
+संभावित रूप से इसे परिवर्तित या पुन: निर्देशित करने के लिए, हमें डेटा एकत्र करने की आवश्यकता होती है।
+परंपरागत रूप से, डेटा संग्रह और नेटवर्क ट्रैफ़िक प्रबंधन को सक्षम करने वाला कोड प्रत्येक एप्लिकेशन के भीतर एम्बेड किया गया था।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:16]:
```suggestion
परंपरागत रूप से, डेटा संग्रह और नेटवर्क ट्रैफ़िक प्रबंधन को सक्षम करने वाला कोड प्रत्येक एप्लिकेशन के भीतर एम्बेड किया जाता है।
```

File: content/hi/service-proxy.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+## यह कैसे मदद करता है
+
+एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को "एक्सटर्नालिजे" करने की अनुमति देती है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:20]: Correct the spelling of 'एक्सटर्नालिजे'

File: content/hi/service-proxy.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+अब इसे ऐप्स के भीतर रहने की आवश्यकता नहीं होगी।
+इसके बजाय, अब इसे प्लेटफ़ॉर्म परत (जहां आपके ऐप्स चलते हैं) में एम्बेड किया जायेगा।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:22]:
```suggestion
इसके बजाय, अब इसे प्लेटफ़ॉर्म की परत (जहां आपके ऐप्स चलते हैं) में एम्बेड किया जायेगा।
```

File: content/hi/service-proxy.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+सर्विस के बीच द्वारपाल के रूप में कार्य करते हुए, प्रॉक्सी यह जानकारी प्रदान करती है कि किस प्रकार का संचार हो रहा है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:24]: Replace the word 'द्वारपाल' with something else.

File: content/hi/service-proxy.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+अपनी अंतर्दृष्टि के आधार पर, वे यह निर्धारित करती है कि किसी विशेष अनुरोध को कहां भेजना है या इसे पूरी तरह से अस्वीकार करना है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:25]: Replace the word 'अंतर्दृष्टि' with something else.

File: content/hi/service-proxy.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+प्रॉक्सी महत्वपूर्ण डेटा एकत्र करती है, रूटिंग का प्रबंधन करती है (सर्विस के बीच ट्रैफ़िक को समान रूप से फैलाना या यदि कुछ सेवाएँ ख़राब हो जाती हैं तो पुनः रूट करना),
+इंक्रिप्टेड कनेक्शन, और कैश मैमोरी (संसाधन खपत को कम करना)।–––
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:28]:
```suggestion
इंक्रिप्टेड कनेक्शन, और कैश मैमोरी (संसाधन की खपत को कम करना और आदि)।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some corrections needs to be done here!

File: content/hi/service-proxy.md (hunk 8 above)
Comment (Aryan-Singla) [reply content/hi/service-proxy.md]: Changed the spelling

File: content/hi/service-proxy.md (hunk 11 above)
Comment (Aryan-Singla) [reply content/hi/service-proxy.md]: update the 'अंतर्दृष्टि' with the 'निरीक्षण'

File: content/hi/service-proxy.md (hunk 5 above)
Comment (Aryan-Singla) [reply content/hi/service-proxy.md]: Updated

File: content/hi/service-proxy.md (hunk 7 above)
Comment (Aryan-Singla) [reply content/hi/service-proxy.md]: updated

**Review by Aryan-Singla — COMMENTED** [review]
I have made the correction as per the suggestion

File: content/hi/service-proxy.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+## यह समस्या का समाधान करता है
+
+सर्विस से सर्विस संचार (उर्फ नेटवर्क ट्रैफ़िक) का ट्रैक रखने के लिए और
+संभावित रूप से इसे परिवर्तित या पुन: निर्देशित करने के लिए, हमें डेटा एकत्र करने की आवश्यकता होती है।
+परंपरागत रूप से, डेटा संग्रह और नेटवर्क ट्रैफ़िक प्रबंधन को सक्षम करने वाला कोड प्रत्येक एप्लिकेशन के भीतर एम्बेड किया जाता है।
+
+## यह कैसे मदद करता है
+
+एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को "एक्सटेरनलिज़" करने की अनुमति देती है।
```
Comment (aj11anuj) [inline content/hi/service-proxy.md:20]:
```suggestion
एक सर्विस प्रॉक्सी हमें इस कार्यक्षमता को लागू करने की अनुमति देती है।
```

**Review by aj11anuj — APPROVED** [review]
lgtm

**Review by kumarankit999 — APPROVED** [review]
/lgtm 
Thanks for raising this PR. Also, a huge shoutout to @aj11anuj for the above review. Looks great in hi

**Comment by bishal7679** [comment]
Thanks a lot @kumarankit999 & @aj11anuj for reviewing throughout
Would love to have it for final review and merge it soon...
/hold

**Comment by Aryan-Singla** [comment]
@bishal7679 Please check this PR pending from long time

**Comment by bishal7679** [comment]
> @bishal7679 Please check this PR pending from long time

@Aryan-Singla really apologize for the delay! will take a look and merge :+1:

File: content/hi/service-proxy.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+status: Completed
+category: टैकनोलजी
+tags: ["नेटवर्किंग"]
+---
+
+सर्विस प्रॉक्सी किसी दिए गए [सर्विस](/service/) से या उससे आने वाले ट्रैफ़िक को रोकता है,
+उसमे फिर कुछ नियम लागू करता है, फिर उस ट्रैफ़िक को किसी अन्य सर्विस पर आगे भेजता है।
+यह अनिवार्य रूप से एक "मध्यस्थ" के रूप में कार्य करता है जो नेटवर्क ट्रैफ़िक के बारे में जानकारी एकत्र करता है और उस पर नियम लागू करता है।
+
+## यह समस्या का समाधान करता है
```
Comment (bishal7679) [inline content/hi/service-proxy.md:12]:
```suggestion
## समस्या
```

File: content/hi/service-proxy.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+
+सर्विस से सर्विस संचार (उर्फ नेटवर्क ट्रैफ़िक) का ट्रैक रखने के लिए और
+संभावित रूप से इसे परिवर्तित या पुन: निर्देशित करने के लिए, हमें डेटा एकत्र करने की आवश्यकता होती है।
+परंपरागत रूप से, डेटा संग्रह और नेटवर्क ट्रैफ़िक प्रबंधन को सक्षम करने वाला कोड प्रत्येक एप्लिकेशन के भीतर एम्बेड किया जाता है।
+
+## यह कैसे मदद करता है
```
Comment (bishal7679) [inline content/hi/service-proxy.md:18]:
```suggestion
## समाधान
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]

**Review by bishal7679 — APPROVED** [review]

**Pull request merged**